test/suites/wdio.conf.js

```diff
@@ -108,13 +107,14 @@ exports.config = {
 	],
 
 	// See also: http://mochajs.org
+	// mochaOpts not populating to browser.options
```

Comment:
Does it still properly configure mocha? If not, the whole thing should be removed.

The comment that it does not populate to `browser.options` can be removed as future readers will not expect anything to populate to `browser.options`